### PR TITLE
GitHub manual reviewers

### PR DESCRIPTION
**Pull request by kateriley1**
Documentation has been updated for requiring manual review before deployment to production.

### CODE REVIEW

**Comment by MattSutton17** [comment]
Not sure I follow quite how the environments are used. I get how you set one up, and I get there's a separate environment for staging and production, but not sure how it would work in practice to deploy to production. What is the trigger for the deployment to production to run?

File: dashboard_publication.md [hunk 1]
```diff
@@ -166,11 +167,33 @@ df = pd.read_csv(response_content)
 
 ---
 
-### References
+## Setting up GitHub manual reviewers for deployment
```
Comment (adrianclay) [inline dashboard_publication.md:170]: I like the action oriented title, it helps me easily understand "Is this the section I wanted to read?".

However, I'm a little confused as to the expectation with these instructions.  Is the idea that you've started deploying to staging using the template, and now want to add a second environment that is manually reviewed before deploying?

Could a similar approach be taken for the S3 section above? Doesn't block this PR.

File: dashboard_publication.md [hunk 2]
```diff
@@ -166,11 +167,33 @@ df = pd.read_csv(response_content)
...
+## References
 https://docs.cloud.service.gov.uk/deploying_services/s3/#bind-an-aws-s3-bucket-to-your-app
 
 https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html
 
+https://cloudlumberjack.com/posts/github-actions-approvals/ 
+
 ---
 
 ## Useful Cloud Foundry commands
```
Comment (adrianclay) [inline dashboard_publication.md:199]: Minor, I wonder if these belong in a separate file.  They feel like they're aimed at a different audience.  One that already knows what they want to achieve, but have forgotten the specific command.  This feels quite different to the audience above.

File: dashboard_publication.md [hunk 3]
```diff
@@ -166,11 +167,33 @@ df = pd.read_csv(response_content)
...
+```
+jobs:
+  deploy-production:
+    name: 'Deploy to production Gov PaaS'
+    runs-on: ubuntu-20.04
+    environment: '<environment_name>'
+    concurrency: production_environment
+    needs: [deploy-staging]
+```
+4. From the GitHub repository click ```Settings```, then ```Environments```, click on the environemnt to configure and click ```Add secret```.
```
Comment (robertblincoe) [inline dashboard_publication.md:184]: environment - typo

File: dashboard_publication.md (hunk 2 above)
Comment (RhysJones125) [reply dashboard_publication.md]: Yeah that seems fair

File: dashboard_publication.md (hunk 1 above)
Comment (RhysJones125) [reply dashboard_publication.md:169]: We've given it a go, hopefully it's a bit more action oriented now

**Pull request merged**